src/lib/components/mods/ModGrid.svelte

```diff
@@ -45,9 +45,19 @@
   let totalMods: number;
 
   let searchField = search;
+  let searchDisabled = true;
+  let searchButtonClass = 'variant-filled-primary';
+  const showPagination: boolean = $mods && !$mods.error && !$mods.fetching;
```

Comment:
$mods is a store, so `showPagination` should be reactive: `$: showPagination = $mods && !mods.error && !mods.fetching`